src/view/treeNodes/pullRequestNode.ts

```diff
@@ -274,7 +274,7 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 		const labelPrefix = currentBranchIsForThisPR ? '✓ ' : '';
 		const tooltipPrefix = currentBranchIsForThisPR ? 'Current Branch * ' : '';
 		const formattedPRNumber = number.toString();
-		const label = `${labelPrefix}#${formattedPRNumber}: ${isDraft ? '[DRAFT] ' : ''}${title}`;
```

Comment:
Hi @alexr00,

I noticed that the PR numbers are no longer displayed in the tree view.
I skimmed the discussion in #681 and I didn't find the change addressed explicitly.
So I was wondering if they were removed for a reason or it's just something that slipped through among the other experiments and changes.

Best!

Reply:
@dippi I removed them because they added a lot of visual noise to the PRs tree view and I didn't think they were very useful. They also took space away from the title of the PR, which is more important than the number.